### PR TITLE
Move validation, install and uninstall scripts from case to samples repo

### PR DESCRIPTION
**Pull request by Kinshuk1993**
Move validation, install and uninstall scripts from case to samples repo

- Move the install script from case repo to samples repo
- Call the install script in samples repo from case repo
- Move the uninstall script from case repo to samples repo
- Call the uninstall script in samples repo from case repo
- Move the validation script from case repo to samples repo
- Call the validation script in samples repo from case repo
- Test 1-click install with refactored install.sh
- Test uninstall script on an existing workspace/installation
- Test uninstall script on new (installed using above) workspace/installation

### CODE REVIEW

**Review by DanRoseus — APPROVED** [review]
Some minor doc comments

**Comment by Kinshuk1993** [comment]
Updated the code with a proper error message for missing params:

- Validation script:


```
./products/bash/1-click-pre-validation.sh -p '' -r '' -u '' -d ''
❌ ERROR: Default admin password is empty. Please provide a value for '-p' parameter.
❌ ERROR: Platform navigator replica count is empty. Please provide a value for '-r' parameter.
❌ ERROR: Default admin username is empty. Please provide a value for '-u' parameter.
❌ ERROR: Demo preparation parameter is empty. Please provide a value for '-d' parameter.

-------------------------------------------------------------------------------------------------------------------

Usage: ./products/bash/1-click-pre-validation.sh -p <csDefaultAdminPassword> -r <navReplicaCount> -u <csDefaultAdminUser> -d <demoPreparation>

-------------------------------------------------------------------------------------------------------------------
```

- Install script:


```
 ./products/bash/1-click-install.sh -n "" -r "" -c "" -u "" -p '' -d '' -a '' -f '' -e '' -h '' -o '' -m '' -q ''
❌ ERROR: 1-click installation namespace is empty. Please provide a value for '-n' parameter.
❌ ERROR: Platform navigator replica count is empty. Please provide a value for '-r' parameter.
❌ ERROR: Default admin username is empty. Please provide a value for '-u' parameter.
❌ ERROR: Demo preparation parameter is empty. Please provide a value for '-d' parameter.
❌ ERROR: Common service password change parameter is empty. Please provide a value for '-c' parameter.
❌ ERROR: Default admin password is empty. Please provide a value for '-p' parameter.
❌ ERROR: Event enabled insurance parameter is empty. Please provide a value for '-a' parameter.
❌ ERROR: Driveway dent deletion parameter is empty. Please provide a value for '-f' parameter.

-------------------------------------------------------------------------------------------------------------------

Usage: ./products/bash/1-click-install.sh -n <NAMESPACE> -r <navReplicaCount> -c <pwdChange> -u <csDefaultAdminUser> -p <csDefaultAdminPassword> -d <demoPreparation> -a <eventEnabledInsuranceDemo> -f <drivewayDentDeletionDemo> -e <demoAPICEmailAddress> -h <demoAPICMailServerHost> -o <demoAPICMailServerPort> -m <demoAPICMailServerUsername> -q <demoAPICMailServerPassword>

-------------------------------------------------------------------------------------------------------------------
```

- Uninstall script:


```
./products/bash/1-click-uninstall.sh -n ''
❌ ERROR: 1-click uninstall namespace is empty. Please provide a value for '-n' parameter.

-------------------------------------------------------------------------------------------------------------------

Usage: ./products/bash/1-click-uninstall.sh -n <NAMESPACE>

-------------------------------------------------------------------------------------------------------------------
```

File: products/bash/1-click-pre-validation.sh [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+  esac
+done
+
+if [[ -z "${csDefaultAdminPassword// }" \
+  || -z "${navReplicaCount// }" \
+  || -z "${csDefaultAdminUser// }" \
+  || -z "${demoPreparation// }" ]]; then
+  echo -e "$cross ERROR: Mandatory parameters are empty"
+  usage
+fi
```
[the remark this answers is not in the document]
Comment (Kinshuk1993) [reply products/bash/1-click-pre-validation.sh]: Done

File: products/bash/1-click-uninstall.sh [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      ;;
+    \? ) usage;
+      ;;
+  esac
+done
+
+if [[ -z "${NAMESPACE// }" ]]; then
+  echo -e "$cross ERROR: A mandatory parameter namespace is empty"
+  usage
+fi
```
[the remark this answers is not in the document]
Comment (Kinshuk1993) [reply products/bash/1-click-uninstall.sh]: Done

File: products/bash/1-click-install.sh [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+  || -z "${pwdChange// }" \
+  || -z "${csDefaultAdminUser// }" \
+  || -z "${csDefaultAdminPassword// }" \
+  || -z "${demoPreparation// }" \
+  || -z "${eventEnabledInsuranceDemo// }" \
+  || -z "${drivewayDentDeletionDemo// }" ]]; then
+  echo -e "$cross ERROR: Mandatory parameters are empty"
+  divider
+  usage
+fi
```
[the remark this answers is not in the document]
Comment (Kinshuk1993) [reply products/bash/1-click-install.sh]: Done

**Review by DanRoseus — APPROVED** [review]
Looks good

**Pull request merged**